src/main/scala/nl/vroste/zio/amqp/model/AMQPConfig.scala

```diff
@@ -0,0 +1,33 @@
+package nl.vroste.zio.amqp.model
+import java.net.URI
+
+case class AMQPConfig(
+  user: String,
+  password: String,
+  vhost: String,
+  heartbeatInterval: Short,
```

Comment:
Shall we make this a zio Duration so we know what the unit is (now I'm not sure if it's seconds or milliseconds)

Reply:
Yes , I'll do that for all time related fields